src/js/simpleSmtpServer.js

```diff
@@ -0,0 +1,84 @@
+// A module to wire in simplesmtp to handle incoming messages.
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.test.mail.smtp.simpleSmtpServer");
+
+var simplesmtp = require("simplesmtp");
+var fs         = require("fs");
+
+gpii.test.mail.smtp.simpleSmtpServer.handleStartData = function (that, connection) {
+    var timestamp = (new Date()).getTime();
+    that.options.messageFile = that.options.config.outputDir + "/message-" + timestamp + ".txt";
```

Comment:
Writing directly into a component's options from code will be prohibited. Either write an "expander" for this purpose or else you can just write it into the component's top level